### DISCUSSION TITLE
Issue with Windows installer on 1.2.0 Beta 5

### DISCUSSION THREAD

**Discussion by jaacMusic** [apohl79/audiogridder#718] [Help] [answered]
There is an issue with the windows installer trying to copy/rename the aaxplugin file when installing. The only option is to skip the file. I ran the installer as admin and I still go the same issue.
![image](https://user-images.githubusercontent.com/68957086/134081442-a336bf72-2ed3-48fb-998e-28a95ea37d08.png)

**Comment by Analog4Lyfe** [comment]
What I did was remove the current plugins out audio / plugins folder. 

Installed. 

Then drag each file into the corresponding folder manual and all worked.

**Comment by apohl79** [comment]
Thx for reporting, need to take look.

**Comment by apohl79** [comment, marked as answer]
I can confirm this error happens, when installing beta5 over an existing plugin installation. The solution is to manually uninstall any version between 1.1.1 to 1.2.0-beta4 before installing beta5 (and later versions).
